pkg/segment/writer/segwriter.go

```diff
@@ -360,8 +360,9 @@ func updateValuesFromConfig() {
 func timeBasedWIPFlushToFile() {
 	for {
 		sleepDuration := time.Duration(config.GetSegFlushIntervalSecs()) * time.Second
```

Comment:
we should also rename the old segflush in lieu of introducing this new variable.